interfaces/Portalicious/tailwind.config.ts

```diff
@@ -10,6 +10,7 @@ export default {
     fontFamily: {
       display: ['Montserrat', 'sans-serif'],
       body: ['"Open Sans"', 'sans-serif'],
+      mono: ['monospace'],
```

Comment:
```suggestion
```

I don't see this in the design system and I would avoid adding something extra that we need to maintain and/or enforce consistency on if it wasn't asked of us.

Reply:
I'd like to add features that help users, regardless if it is asked of me.

So, I tried initially to use the "Tailwind default font-stacks", but those where probably overridden by this config, so I added the bare-minimum to start with. And at some point other, more specific decisions could be added...

Reply:
> I'd like to add features that help users, regardless if it is asked of me.

I'm not against that per se, but I see it as a problem when a dev introduces a design change, without input from design/PM. The assumption that "this helps users" is not one that they are necessarily on board with. And the risk of a design change made at this stage in the process is that we either
- End up having to revert the change because design / PM noticed it at a later stage or 
- A developer reverts this change because they notice it and it's a discrepancy with design (this could be me)

If you feel strongly about the monospace being necessary then I would do a couple of things
- apply it to the p-password in styles.scss rather than throughout the codebase (so we only do it one place, and it's easy to revert if we need to)
- add a note that we have done this in the "design review" task for this PBI, so we can revert if we need to